test/e2e/nodepool_test.go

```diff
@@ -115,11 +115,11 @@ func TestNodePool(t *testing.T) {
 					},
 					{
 						name: "TestNodePoolPrevReleaseN1",
-						test: NewNodePoolPrevReleaseCreateTest(hostedCluster, globalOpts.N1MinorReleaseImage, clusterOpts),
+						test: NewNodePoolPrevReleaseCreateTest(hostedCluster, globalOpts.N1MinorReleaseImage, clusterOpts, true),
 					},
 					{
 						name: "TestNodePoolPrevReleaseN2",
```

Comment:
with the policy being proposed in this PR, wouldn't this depend if the hc version is even or odd?

Reply:
Even versions support N-2, while odd versions do not.
So, if hc is an even version:
```
{
	name: "TestNodePoolPrevReleaseN1",
	test: NewNodePoolPrevReleaseCreateTest(hostedCluster, globalOpts.N1MinorReleaseImage, clusterOpts, true),
},
{
	name: "TestNodePoolPrevReleaseN2",
	test: NewNodePoolPrevReleaseCreateTest(hostedCluster, globalOpts.N2MinorReleaseImage, clusterOpts, true),
}
```
But if hc is an odd version, it should be:
```
{
	name: "TestNodePoolPrevReleaseN1",
	test: NewNodePoolPrevReleaseCreateTest(hostedCluster, globalOpts.N1MinorReleaseImage, clusterOpts, true),
},
{
	name: "TestNodePoolPrevReleaseN2",
	test: NewNodePoolPrevReleaseCreateTest(hostedCluster, globalOpts.N2MinorReleaseImage, clusterOpts, false),
}
```